### PR TITLE
Updated Facebook presence

### PR DESCRIPTION
**Pull request by 7loud**
Proof:
Screenshot_1

Screenshot_2

Screenshot_3

image
etc.

### CODE REVIEW

File: websites/F/Facebook/dist/metadata.json [hunk 1]
```diff
@@ -17,7 +21,7 @@
     "zh_CN": "与朋友、家人和其他你认识的人联系。分享照片和视频、发送消息和获取更新。"
   },
   "url": "www.facebook.com",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (EncryptedDev) [inline websites/F/Facebook/dist/metadata.json:24]:
```suggestion
  "version": "3.0.0",
```

File: websites/F/Facebook/presence.ts [hunk 2]
```diff
@@ -212,6 +235,73 @@ presence.on("UpdateData", async () => {
...
+    else if (document.location.pathname.includes("/events/notifications"))
+      presenceData.state = "Notifications";
+    else if (document.location.pathname.includes("/events/create"))
+      presenceData.state = "Creating event";
+    else if (document.location.pathname.includes("/events/search")) {
+      if (!privacyMode) {
+        presenceData.details = "Events - Search";
+        presenceData.state = document.querySelectorAll(
+          "label.rq0escxv.a8c37x1j.a5nuqjux.l9j0dhe7.k4urcfbm > input.oajrlxb2.rq0escxv.f1sip0of.hidtqoto[type='search']"
+        )?.[1]?.value;
```
Comment (EncryptedDev) [inline websites/F/Facebook/presence.ts:285]: I could be wrong, but `value` doesn't exist on `Element`

File: websites/F/Facebook/presence.ts [hunk 3]
```diff
@@ -77,20 +80,37 @@ presence.on("UpdateData", async () => {
...
+        document.querySelector("span.nc684nl6 > span")?.textContent ||
+        document.querySelector("span.nc684nl6 > a > strong > span")
+          ?.textContent;
 
       presenceData.details = "Viewing a photo on:";
       presenceData.state = `${username}'s profile'`;
+
+      presenceData.buttons = [
+        {
+          label: `View photo`,
```
Comment (EncryptedDev) [inline websites/F/Facebook/presence.ts:110]:
```suggestion
          label: "View photo",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/F/Facebook/presence.ts (hunk 2 above)
Comment (N0chteil) [reply websites/F/Facebook/presence.ts:285]: I've tested it, it works

**Review by Bas950 — DISMISSED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**